ee/clickhouse/sql/person.py

```diff
@@ -230,7 +230,7 @@
 """
 
 INSERT_COHORT_ALL_PEOPLE_THROUGH_PERSON_ID = """
-INSERT INTO {cohort_table} SELECT generateUUIDv4(), id, %(cohort_id)s, %(team_id)s, %(_timestamp)s, 0 FROM (
+INSERT INTO {cohort_table} SELECT generateUUIDv4(), person_id, %(cohort_id)s, %(team_id)s, %(_timestamp)s, 0 FROM (
```

Comment:
This has a bug, but I guess this query is not used anyway since you can't save funnel persons as a cohort.

Reply:
yes but... is it a mistake now? did this fix something? is there a real mistake in production and that also doesn't work?

Reply:
The right test should be able to verify this, right?

Reply:
I just uncommented a test that was [previously commented out](https://github.com/PostHog/posthog/pull/5364). There was a [recent refactor](https://github.com/PostHog/posthog/pull/5842/files#diff-8c1a8fdf464c1e77c60c9d3a113fa05e04ba342aa680324f9d037e936f3dfd6cL262) that changed how the query worked and since the test was commented out, the error wasn't caught.